MaxText/checkpointing.py

```diff
@@ -59,6 +59,8 @@ def create_orbax_checkpoint_manager(
   else:
     item_names = ("items",)
 
+  # local storage checkpoint needs parent directory created
+  p.mkdir(exist_ok=True, parents=True)
```

Comment:
Could you double check if this line support cloud gcs bucket as well?

Maybe we can use
[`tf.io.gfile.mkdir`](https://www.tensorflow.org/api_docs/python/tf/io/gfile/mkdir) which supports both gcs and local directory.

Reply:
this is `epath.Path` and I used it before with mkdir, it seems to work on gcs, I believe it's using gfile under the hood